### PR TITLE
Ext mon detector updates

### PR DESCRIPTION
**Pull request by fine1-sam**
Add trigger scintillators to ext mon detector

### CODE REVIEW

**Comment by kutschke** [comment]
@FNALbuild run build test

**Comment by fine1-sam** [comment]
@FNALbuild run build test

**Comment by fine1-sam** [comment]
@FNALbuild run build test

**Comment by kutschke** [comment]
Thanks Sam.  I will wait for Andrei to approve.

**Comment by kutschke** [comment]
@gaponenko a gentle reminder that this PR is on your list of PRs to review.

File: Mu2eG4/geom/extmon_fnal_v02.txt [hunk 1]
```diff
@@ -175,6 +175,15 @@ string         extMonFNAL.planeMaterial = "G4_C";
 bool extMonFNAL.stackMotherVisible = true;
 bool extMonFNAL.stackMotherSolid = true;
 
+//================================================================
+// Scintillator parameters
+
+vector<double> extMonFNAL.scintHalfSize = { 54.0 , 42.0, 3.0 };
+string         extMonFNAL.scintMaterial = "G4_C";
```
Comment (gaponenko) [inline Mu2eG4/geom/extmon_fnal_v02.txt:182]: The material here should be a scintillator, not just carbon.

File: Mu2eG4/geom/extmon_fnal_v02.txt [hunk 2]
```diff
@@ -175,6 +175,15 @@ string         extMonFNAL.planeMaterial = "G4_C";
...
+double         extMonFNAL.scintOffset = 29.0;
+double         extMonFNAL.scintInnerOffset = 33;
+double         extMonFNAL.scintGap = 5.3;
```
Comment (gaponenko) [inline Mu2eG4/geom/extmon_fnal_v02.txt:185]: please add comments explaining the meaning of these parameters

File: Mu2eG4/inc/constructExtMonFNAL.hh [hunk 3]
```diff
@@ -59,6 +59,16 @@ namespace mu2e {
                                   bool const placePV
                                   );
 
+  void constructExtMonFNALScintillators(const VolumeInfo& mother,
+                                        const ExtMonFNALModule& module,
```
Comment (gaponenko) [inline Mu2eG4/inc/constructExtMonFNAL.hh:63]: Why does it need the module argument?

**Review by gaponenko — CHANGES_REQUESTED** [review]
Hi Sam, please see the comments below.

**Comment by fine1-sam** [comment]
@FNALbuild run build test

**Comment by fine1-sam** [comment]
@gaponenko I incorporated your changes. Let me know what you think. In particular, there may be a clearer and more concise set of comments than the ones I wrote to specify which offsets correspond to which scintillators.
Thanks,
Sam

File: Mu2eG4/geom/extmon_fnal_v02.txt [hunk 4]
```diff
@@ -179,12 +179,11 @@ bool extMonFNAL.stackMotherSolid = true;
 // Scintillator parameters
 
 vector<double> extMonFNAL.scintHalfSize = { 54.0 , 42.0, 3.0 };
-string         extMonFNAL.scintMaterial = "G4_C";
-double         extMonFNAL.scintOffset = 29.0;
-double         extMonFNAL.scintInnerOffset = 33;
-double         extMonFNAL.scintGap = 5.3;
+string         extMonFNAL.scintMaterial = "Scintillator";
+double         extMonFNAL.scintOffset = 29.0; //Distance from pixel plane center to center of scintillator on outter ends of detector table
```
Comment (gaponenko) [inline Mu2eG4/geom/extmon_fnal_v02.txt:183]: Do you mean the scintillator closest to the pixels?

**Comment by kutschke** [comment]
Just checking in.  I think that this is waiting for Sam to answer one question from Andrei about updating a comment.

File: Mu2eG4/geom/extmon_fnal_v02.txt (hunk 4 above)
Comment (fine1-sam) [reply Mu2eG4/geom/extmon_fnal_v02.txt:183]: Yes I do, in particular the scintillator closest to the pixels that is attached to the double mounted scintillator setup.

**Comment by kutschke** [comment]
@FNALbuild run build test

**Review by gaponenko — APPROVED** [review]
Sam, thank you!

**Comment by gaponenko** [comment]
@FNALbuild run build tests

**Comment by kutschke** [comment]
This is ready to merge.  There are other PRs that might also be ready today and there is one that we want to merge by itself so that the nightly validation is a clean test of just that PR.  I expect this will be merged today or tomorrow depending on the state of the other PRs.

**Pull request merged**